modules/wmco-disconnected-cluster.adoc

```diff
@@ -4,25 +4,31 @@
 
 :_mod-docs-content-type: CONCEPT
 [id="wmco-disconnected-cluster_{context}"]
-= Using Windows containers in a disconnected cluster
```

Comment:
I prefer this being framed `Using Windows containers in a disconnected cluster`

Reply:
@saifshaikh48 As I understand, folks in connected clusters can user repository mirroring also. That is why I wanted to emphasize mirroring.